st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -1,272 +0,0 @@
-# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Yeah we need to verify the symlink works as expected end to end (it might cause issues with virtual environments...).

One option would also be to have a Python module here which basically just imports everything from new location and re-exports it, so:

```python
__all__ = [
    'Action'
]


from st2actions.runners.base_action import Action
```

This is probably also better from cross-OS compatibility perspective :)